oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -200,6 +218,16 @@ String createTokenUrlWithScopes() {
     if (!scopes.isEmpty()) {
       tokenUrl.set("scopes", Joiner.on(',').join(scopes));
     }
+    if (transport == Transport.MTLS) {
+      tokenUrl.set("transport", "mtls");
```

Comment:
You can add fields to the Enum class, so that each Enum can have its own label, then you can just do `tokenUrl.set("transport", transport.getLabel())`. See this [example](https://www.baeldung.com/java-enum-values#adding-constructor).

Reply:
Thanks for pointing this out! Done in latest commit.